### PR TITLE
new blog and layout changes to support display

### PR DESCRIPTION
**Pull request by nitindermohan**
Added Accordion blog article.

Also added layout changes in webpage to make it render better.

### CODE REVIEW

**Comment by nitindermohan** [comment]
This PR can be reviewed now

File: content/contributors/_index.md
Comment (giobart) [inline content/contributors/_index.md:1]: this is not showing rn right?

Comment (nitindermohan) [reply content/contributors/_index.md:1]: if you make draft: true; then the blog article wont show. We only want to keep this as example. 

So yes, this doesn't show up

Comment (nitindermohan) [reply content/contributors/_index.md:1]: draft: true hides the blog. This blog will only be used for example purposes

Comment (giobart) [reply content/contributors/_index.md:1]: But I meant this contirbutors section. This is not showing in the nav rn.

Comment (nitindermohan) [reply content/contributors/_index.md:1]: Yes. There is some issue with the contributor page. I want to mark this as a bug and handle this when I find time.

Ideally I want contributor page to look like this: https://getdoks.org/contributors/henk-verlinde/
which maps to the changes here [content/contributors/henk-verlinde](https://github.com/thuliteio/getdoks.org/tree/main/content/contributors/henk-verlinde)

**Review by giobart — APPROVED** [review]

File: content/contributors/_index.md
Comment (nitindermohan) [reply content/contributors/_index.md:1]: An issue from this discussion now exists https://github.com/nitindermohan/oak-new-docs/issues/41

**Pull request merged**